pkg/list/format/delimited.go

```
@@ -0,0 +1,119 @@
+			// collect all keys from the map
+			for k := range stackData {
+				valueKeys = append(valueKeys, k)
+			}
+			break
+		}
+	}
+	sort.Strings(valueKeys)
+
+	// Create header and rows we may need to change this in the future to be more flexible
+	header := []string{"Key"}
+	for _, k := range keys {
```

golangci-lint (1.62.2)

S1011: should replace loop with `header = append(header, keys...)`

(gosimple)